app/components/calls_to_action/apply_block_component.rb

```diff
@@ -0,0 +1,22 @@
+module CallsToAction
+  class ApplyBlockComponent < ViewComponent::Base
+    attr_reader :title, :image, :title_color, :text, :link_text, :link_target
+
+    def initialize(
+      title: "Start your application",
+      image: "apply-block-promo.jpg",
+      title_color: "green",
+      text: "Create an account and start your application for a teacher training course.",
+      link_text: "Apply for a course",
```

Comment:
```suggestion
      link_text: "Apply for teacher training",
```
@emmaachesongray to confirm